src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private StringBuilder stringBuilder;
+    private String resultString;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Method is too long and performs several tasks. Please split logic to 3 private methods (read data from file, process data, write report to file) and call them in public one